apps/src/templates/instructions/codeReviewV2/CodeReviewTimelineElement.jsx

```diff
@@ -15,23 +16,25 @@ export const codeReviewTimelineElementType = {
 // 1. CREATED - this is the node that represents the first element on the timeline, it has a line extending below
 //    it unless it is also the only element in the timeline. Otherwise this node does not vary from project to project.
 // 2. COMMIT - this represents a commit to the project. It will display an eyeball on the left which links to the
-//    project version if the project version is present and not expired. It also handles a child which represents
+//    project version if the project version is present. It also handles a child which represents
 //    the specific content in the commit. If it is the last element in the timeline, it will not have a line extending
 //    down from it.
 // 3. CODE_REVIEW - this represnts a code review request and comments made. It will display an eyeball to the left
-//    which links to the project version if it is present and not expired. It takes a child which should be the actual
+//    which links to the project version if it is present. It takes a child which should be the actual
 //    code review component. If it is the last element in the timeline it will not have a line extending below it.
 const CodeReviewTimelineElement = ({
   type,
   isLast,
   projectVersionId,
-  isProjectVersionExpired,
+  viewAsCodeReviewer,
   children
 }) => {
-  const displayVersion = !isProjectVersionExpired && !!projectVersionId;
   const versionLink =
     location.origin + location.pathname + '?version=' + projectVersionId;
 
+  // You can only see previous versions of your own project
+  const displayEyeball = !viewAsCodeReviewer && !!projectVersionId;
```

Comment:
Yeah, I can't find the thread where we talked about it, so maybe we chatted in a meeting, but it was decided that a student shouldn't be allowed to view the version history of another student. That's a good point James, that maybe the timeline UI doesn't make sense in that case. Or we allow student's to view older versions of another student's code.

Reply:
@moneppo I'll take this opportunity to say that, in the future for a feature like this, we should have separate mocks for each "role" (in this case, code review owner, peer, and teacher).  I think it would help us discover these types of issues earlier in the process and reduce the overall development cost.